pages/common/doctl-apps.md

```diff
@@ -0,0 +1,36 @@
+# doctl apps
+
+> Used to manage digitalocean apps.
+> More information: <https://docs.digitalocean.com/>.
+
+- Create an app:
+
+`doctl apps create`
+
+- Create a deployment:
+
+`doctl apps create-deployment`
+
+- Delete an app interactively:
+
+`doctl apps delete`
```

Comment:
How does it know which app to delete? According to https://docs.digitalocean.com/reference/doctl/reference/apps/delete/, you need this:
```suggestion
`doctl apps delete {{app_id}}`
```